.github/workflows/ci.yml

```diff
@@ -355,38 +355,68 @@ jobs:
             **/surefire-reports/TEST-*.xml
           retention-days: ${{ env.TEST_REPORT_RETENTION_DAYS }}
 
+  build-test-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # checkout all commits to be able to determine merge base for GIB
+      - name: Fetch base ref
+        run: |
+          [ "$GITHUB_REF_NAME" != master ] && git fetch --no-tags --prune --progress --no-recurse-submodules origin "$GITHUB_BASE_REF"
```

Comment:
Remove `--progress --no-recurse-submodules`

Reply:
Make this a single `run:` line

Reply:
Done! Sorry about this, I missed it during rebase.